Content/Presentations/Fortran/KokkosTutorial_Fortran.tex

```diff
@@ -0,0 +1,240 @@
+
+\input{KokkosTutorial_PreTitle}
+
+\title{Kokkos-Fortran-Interop}
+
+\input{KokkosTutorial_PostTitle}
+
+\usepackage{minted}
+
+\begin{document}
+
+\begin{frame}
+	\titlepage
+\end{frame}
+
+\begin{frame}{Installation}
```

Comment:
Should we make some comment that the tool is not really actively developed?

Reply:
No, the plan is to get the library back into shape.

Reply:
Ok, that is interesting. Ok, we can keep it as it is.

Reply:
Resloved